src/server/interfaces.ts

```diff
@@ -20,6 +20,7 @@ export interface IRunnerHandlebarsContext {
 
     origin: string;
     host: string;
+    assets: any;
```

Comment:
Actually, wait, it looks like it's not a string-to-string but rather a string-to-object map.  In that case it should be { [key: string]: any }

---
In reply to: [126537988](https://github.com/OfficeDev/script-lab/pull/474#discussion_r126537988) [](ancestors = 126537988)